docs/pages/database-access/guides/azure-redis.mdx

```diff
@@ -0,0 +1,224 @@
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Create the Database Service configuration, specifying a region like this:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+  -o file \
+  --proxy=tele.example.com:443 \
+  --token=/tmp/token \
+  --azure-redis-discovery=eastus
+```
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+  -o file \
+  --proxy=teleport.example.com:3080 \
+  --token=/tmp/token \
+  --azure-redis-discovery=eastus
+```
+</ScopedBlock>
+
+The command will generate a Database Service configuration with Azure Cache for
+Redis auto-discovery enabled in the `eastus` region and place it at the
+`/etc/teleport.yaml` location.
+
+<Admonition type="tip">
+Use `--azure-redis-discovery=*` to discover databases in all regions. In
+addition to the region, you can optionally specify
+`--azure-resource-group=<resource-group-name>`,
+`--azure-subscription=<subscription-id>`, or `--labels=<key>=<value>` to
+further customize the scopes of the auto-discovery.
+</Admonition>
+
+## Step 4/6. Configure IAM permissions for Teleport
+
+The Teleport Database Service needs Azure IAM permissions to:
+
+- Discover and register Azure Cache for Redis databases. 
+- Retrieve Redis access keys to authenticate with the databases.
+
+### Configure Azure service principal
+
+(!docs/pages/includes/database-access/azure-configure-service-principal.mdx!)
+
+### Create a custom role
+
+Now create a custom role with Azure Cache for Redis permissions. For example:
+```json
+{
+    "properties": {
+        "roleName": "TeleportDiscovery",
+        "description": "Allows Teleport to discover Azure Cache For Redis databases and list keys",
+        "assignableScopes": [
+            "/subscriptions/11111111-2222-3333-4444-555555555555"
+        ],
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.Cache/redis/read",
+                    "Microsoft.Cache/redis/listKeys/action",
+                    "Microsoft.Cache/redisEnterprise/read",
+                    "Microsoft.Cache/redisEnterprise/databases/read",
+                    "Microsoft.Cache/redisEnterprise/databases/listKeys/action"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+    }
+}
+```
+
+This role definition allows Teleport to discover and list keys for both Azure
+Redis and Azure Redis Enterprise, but Teleport only needs permissions for the
+database types you have. The assignable scopes include a subscription, so the
+role can be assigned at any resource scope within that subscription, or
+assigned using the subscription scope itself.
+
+<Admonition type="note" title="Custom role assignable scope">
+Custom roles, unlike
+[Azure built-in roles](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles),
```

Comment:
I think we should move this link to a "Further reading" section so the user can get to the end of this guide more quickly. Same with the second link in this Admonition. (per the [style guide](https://goteleport.com/docs/contributing/documentation/style-guide/#expectations-1))